ethereum/attestation-test/contracts/VerifyTicket.sol

```diff
@@ -91,18 +91,16 @@ contract VerifyTicket {
         uint length;
     }
 
+    /**
+    * Perform TicketAttestation verification
+    * NOTE: This function DOES NOT VALIDATE the subject; you must perform validation of the subject from the calling function.
+    **/
     function verifyTicketAttestation(bytes memory attestation, address attestor, address ticketIssuer) external view returns(address subject, bytes memory ticketId, bytes memory conferenceId, bool timeStampValid)
```

Comment:
> I think it makes more sense to simply return a single value indicating whether validation is ok or not, rather than only a boolean indicating whether the timestamp is valid or not

We still need to return the subject, ticket and conference id for any calling contract, do you mean that the boolean should be a global 'pass/fail' of the whole attestation?

Reply:
Yes, sorry, that is exactly what I mean. The Boolean should indicate whether **anything** went wrong

Reply:
Updated